### PR TITLE
Improve readability of htmlentities.xml

### PR DESCRIPTION
**Pull request by morganarnel**
This line now reads better when you're in a hurry (imo), and doesn't give the impression that ALL characters will be converted to entities.

### CODE REVIEW

**Comment by cmb69** [comment]
Thanks for the PR! I wonder whether it would be better to hint at `get_html_translation_table()` instead of referring to the rather vague "have HTML character entity equivalents", and clarifying by mentioning the flags.

**Comment by cmb69** [comment]
Not quite what I had in mind, but looks good to me. Thank you!

File: reference/strings/functions/htmlentities.xml [hunk 1]
```diff
@@ -17,8 +17,7 @@
   </methodsynopsis>
   <para>
    This function is identical to <function>htmlspecialchars</function> in all
-   ways, except with <function>htmlentities</function>, all characters which
-   have HTML character entity equivalents are translated into these entities.
+   ways, except with <function>htmlentities</function>, all characters which have HTML character entity equivalents are translated into these entities. The <function>get_html_translation_table</function> function can be used to return the translation table used dependent upon the provided <parameter>flags</parameter> constants.
```
Comment (Crell) [inline reference/strings/functions/htmlentities.xml:20]:
```suggestion
   ways, except with <function>htmlentities</function>, all characters which have HTML character
   entity equivalents are translated into these entities. The <function>get_html_translation_table</function>
   function can be used to return the translation table used dependent upon the provided
   <parameter>flags</parameter> constants.
```

Text should be wrapped consistently.

Comment (Crell) [inline reference/strings/functions/htmlentities.xml:20]: GitHub isn't letting me do a multi-line suggestion here, but the text should be hard-wrapped consistently.

Comment (cmb69) [inline reference/strings/functions/htmlentities.xml:20]:
```suggestion
   ways, except with <function>htmlentities</function>, all characters which
   have HTML character entity equivalents are translated into these entities.
   The <function>get_html_translation_table</function> function can be used
   to return the translation table used dependent upon the provided
   <parameter>flags</parameter> constants.
```

**Comment by cmb69** [comment]
Thank you!

**Pull request merged**